.github/workflows/linux_wheel.yaml

```diff
@@ -121,7 +123,7 @@ jobs:
           ls
       - name: Smoke test
         run: |
-          python test/decoders/manual_smoke_test.py
+          python -X faulthandler test/decoders/manual_smoke_test.py
```

Comment:
In case we run into any problems again on Linux or Mac, I think running the smoke test with the faulthandler on will be helpful.